lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -151,6 +152,10 @@ def extract_student(student, features):
             for meta_feature, meta_key in meta_features:
                 student_dict[meta_feature] = meta_dict.get(meta_key)
 
+            meta_city = meta_dict.get('city')
+            if include_city_column and meta_city:
+                student_dict['city'] = meta_city
+
```

Comment:
@kaizoku, with this change, wouldn't we have the same value in `student_dict['meta.city']` (populated in the `for` loop above) and `student_dict['city']`?

Reply:
@lgp171188 , only if we include `meta.city` and `city` in the query features. Only "city" is configured by default, if both are manually configured, this might be an issue, but should we be handling that scenario? As it stands, end users won't have to configure any options to get the city, it will always be there. So adding `meta.city` to the `student_profile_download_fields` would in a way be a misconfiguration.